src/components/user-list.js

```diff
@@ -74,30 +41,34 @@ export default class UserList extends React.Component {
 
     render () {
         const { wantAdd, users } = this.state;
+
         return (
             <div className="UserList">
-                <button className="ButtonAddUser" data-testid="ButtonAddUser" onClick={this.handleClickAddUser}>Add new
-                    User
+                <button className="ButtonAddUser" data-testid="ButtonAddUser" onClick={this.handleClickAddUser}>
+                    Add new User
                 </button>
                 {
                     wantAdd &&
                     (
-                        <div>
-                            <User className="NewUser" isNewUser={true}/>
-                            <button className="ButtonAddUser" onClick={this.handleClickSubmit}>Submit</button>
-                        </div>
+                        <User isNewUser={true}/>
                     )
+
                 }
                 <hr/>
                 {
                     users.map((user) => {
+                        const info = {
+                            id:      user.id,
+                            name:    user.name,
+                            email:   user.email,
+                            phone:   user.phone,
+                            website: user.website
+                        };
+
                         return (
                             <div className="UserInner" key={user.id}>
-                                <User name={user.name}
-                                      email={user.email}
+                                <User info={info}
```

Comment:
I'm still wondering about wrappers. Can we get rid of this `div className="UserInner"` wrapped?